### ISSUE TITLE
generate local Rakefile instead of using redstorm bin?

### ISSUE THREAD

**Issue by colinsurprenant** [colinsurprenant/redstorm#27]
see if instead of using the redstorm bin for all commands in workflow, we could use a generator to create a local Rakefile and expose rake tasks for the workflow. This would allow easy local customization if needed.

**Comment by palanglung** [comment]
Good idea i think especially if we use a deployment strategy like capistrano
